### PR TITLE
Get rid of a couple of baseroms

### PR DESCRIPTION
**Pull request by DizzyEggg**
Booooooorriiiinggg

### CODE REVIEW

File: data/graphics.s [hunk 1]
```diff
@@ -1197,8 +1197,44 @@ gUnknown_08DC9608:: @ 8DC9608
...
+    
+	.space 0x120
 
+	.align 2
+gBerryBlenderCountdownNumbersTiles:: @ 8DC9F88
+	.incbin "graphics/berry_blender/countdown_numbers.4bpp"
+    
+	.align 2
+gBerryBlenderStartTiles:: @ 8DCA588
+	.incbin "baserom.gba" , 0xdca588, 1024
```
Comment (Diegoisawesome) [inline data/graphics.s]: Why wasn't this extracted?

File: data/graphics.s [hunk 2]
```diff
@@ -1197,8 +1197,44 @@ gUnknown_08DC9608:: @ 8DC9608
...
+    
+    .space 512
```
Comment (Diegoisawesome) [inline data/graphics.s]: .space 0x200

File: data/item_menu.s [hunk 3]
```diff
@@ -4,61 +4,100 @@
...
+	.4byte gMenuText_Use, ItemMenu_UseInBattle
+	.4byte gMenuText_Check, ItemMenu_UseOutOfBattle
+	.4byte gMenuText_Walk, ItemMenu_UseOutOfBattle
+	.4byte gMenuText_Deselect, ItemMenu_Register
+	.4byte gMenuText_CheckTag, ItemMenu_CheckTag
+	.4byte gMenuText_Confirm, unknown_ItemMenu_Confirm
+	.4byte gMenuText_Show, unknown_ItemMenu_Show
+	.4byte gMenuText_Give2, unknown_ItemMenu_Give2
+	.4byte gMenuText_Confirm, unknown_ItemMenu_Confirm2
+    .4byte gText_EmptyString2, 0
```
Comment (Diegoisawesome) [inline data/item_menu.s]: Four spaces -> tab

Comment (Diegoisawesome) [reply data/item_menu.s]: Also when the item is a pointer with value zero, use NULL instead.

File: data/item_menu.s [hunk 4]
```diff
@@ -4,61 +4,100 @@
...
+	.4byte unknown_item_menu_type
+	.4byte item_menu_type_2
+	.4byte display_sell_item_ask_str
+	.4byte unknown_ItemMenu_Confirm
+	.4byte unknown_item_menu_type
+	.4byte display_deposit_item_ask_str
+	.4byte unknown_item_menu_type
+	.4byte unknown_item_menu_type
+	.4byte unknown_item_menu_type
+	.4byte 0
```
Comment (Diegoisawesome) [inline data/item_menu.s]: 0->NULL

File: data/item_menu.s [hunk 5]
```diff
@@ -4,61 +4,100 @@
...
+	.4byte item_menu_type_b
+
+.align 2
```
Comment (Diegoisawesome) [inline data/item_menu.s]: should be indented

File: data/mauville_old_man.s [hunk 6]
```diff
@@ -5,8 +5,8 @@
 	.align 2, 0
 
 gUnknown_0859EFE4:: @ 859EFE4
-	.incbin "baserom.gba", 0x59efe4, 0xc
-
+	.2byte 0x2811, 0x1029, 0x1018, 0xE0D, 0x1A1A, 0x1A1D
+
```
Comment (Diegoisawesome) [inline data/mauville_old_man.s]: Remove these spaces

File: data/save.s [hunk 7]
```diff
@@ -4,4 +4,18 @@
...
+    .2byte 0, 0xF80
+    .2byte 0xF80, 0xF80
+    .2byte 0x1F00, 0xF80
+    .2byte 0x2E80, 0xF80
+    .2byte 0x3E00, 0xF80
+    .2byte 0x4D80, 0xF80
+    .2byte 0x5D00, 0xF80
+    .2byte 0x6C80, 0xF80
+    .2byte 0x7C00, 0x7D0
+
```
Comment (Diegoisawesome) [inline data/save.s]: Remove spaces at end of file, replace other groups of four spaces with tabs

File: data/script_menu.s [hunk 8]
```diff
@@ -797,16 +694,55 @@ gUnknown_0858B760:: @ 858B760
...
+	.4byte gText_BattlePalace
+	.4byte gText_BattleArena
+	.4byte gText_BattlePike
+	.4byte gText_BattlePyramid
 
 gUnknown_0858BB68:: @ 858BB68
-	.incbin "baserom.gba", 0x58bb68, 0x8
+	.byte 0x4A, 0x4B, 0x4C, 0x4D, 0x4E, 0x4F
 
+.align 2
```
Comment (Diegoisawesome) [inline data/script_menu.s]: indent with tab

File: src/berry_blender.c [hunk 9]
```diff
@@ -11,8 +13,585 @@ IWRAM_DATA s16 gUnknown_03000DF8[6];
...
+// unused text?
+static const u8 sUnusedText_YesNo[] = _("YES\nNO");
+static const u8 sUnusedText_2[] = _("▶");
+static const u8 sUnusedText_Space[] = _(" ");
+static const u8 sUnusedText_Terminating[] = _("Terminating.");
+static const u8 sUnusedText_LinkPartnerNotFound[] = _("Link partner(s) not found.\nPlease try again.\p");
+
+const u8 gText_BerryBlenderStart[] = _("Starting up the BERRY BLENDER.\pPlease select a BERRY from your BAG\nto put in the BERRY BLENDER.\p");
+const u8 gText_NewParagraph[] = _("\p");
+const u8 gText_wasMade[] = _(" was made!");
```
Comment (Diegoisawesome) [inline src/berry_blender.c]: Capitalize the W in the symbol name (WasMade)

File: src/berry_blender.c [hunk 10]
```diff
@@ -11,8 +13,585 @@ IWRAM_DATA s16 gUnknown_03000DF8[6];
...
+{
+    sText_Mister,
+    sText_Laddie,
+    sText_Lassie,
+    sText_Master,
+    sText_Dude,
+    sText_Miss
+};
+
+static const u8 sText_PressA_ToStart[] = _("Press the A Button to start.");
```
Comment (Diegoisawesome) [inline src/berry_blender.c]: Remove underscore after A

File: src/berry_blender.c [hunk 11]
```diff
@@ -11,8 +13,585 @@ IWRAM_DATA s16 gUnknown_03000DF8[6];
...
+static const u8 sText_PleaseWaitAWhile[] = _("Please wait a while.");
+const u8 sText_CommunicationStandby[] = _("Communication standby…");
+const u8 sText_WouldLikeToBlendAnotherBerry[] = _("Would you like to blend another BERRY?");
+const u8 sText_RunOutOfBerriesForBlending[] = _("You’ve run out of BERRIES for\nblending in the BERRY BLENDER.\p");
+const u8 sText_YourPokeblockCaseIsFull[] = _("Your {POKEBLOCK} CASE is full.\p");
+const u8 sText_hasNoBerriesToPut[] = _(" has no BERRIES to put in\nthe BERRY BLENDER.");
```
Comment (Diegoisawesome) [inline src/berry_blender.c]: Capitalize H

File: src/berry_blender.c [hunk 12]
```diff
@@ -11,8 +13,585 @@ IWRAM_DATA s16 gUnknown_03000DF8[6];
...
+const u8 sText_someonesPokeblockCaseIsFull[] = _("’s {POKEBLOCK} CASE is full.\p");
```
Comment (Diegoisawesome) [inline src/berry_blender.c]: ApostropheSPokeblockCaseIsFull

File: data/graphics.s (hunk 1 above)
Comment (DizzyEggg) [reply data/graphics.s]: I tried but the graphics didn't match

**Comment by Diegoisawesome** [comment]
```tools/gbagfx/gbagfx graphics/unknown/8339514.pal graphics/unknown/8339514.gbapal
Failed to open JASC-PAL file "graphics/unknown/8339514.pal" for reading.```

File: asm/berry_blender.s [hunk 13]
```diff
@@ -5555,7 +5555,7 @@ _080826EC:
 	adds r1, r0
 	adds r0, r4, 0
 	bl StringCopy
-	ldr r1, =sText_someonesPokeblockCaseIsFull
+	ldr r1, =ApostropheSPokeblockCaseIsFull
```
Comment (Diegoisawesome) [inline asm/berry_blender.s]: You forgot the prefix

**Pull request merged**